README.md

```diff
@@ -1,56 +1,65 @@
+| [modernisation-platform-terraform-bastion-linux](https://github.com/ministryofjustice/modernisation-platform-terraform-bastion-linux)                             | Module for creating Linux bastion servers in member AWS accounts                                                                                                |
+| [modernisation-platform-terraform-ecs](https://github.com/ministryofjustice/modernisation-platform-terraform-ecs)                                                 | Module for creating ECS cluster (Linux/Windows) solely for EC2 launch type                                                                                      |
+| [modernisation-platform-terraform-s3-bucket](https://github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket)                                     | Module for creating S3 buckets with sensible defaults e.g. replication, encryption                                                                              |
+| [modernisation-platform-terraform-s3-bucket-replication-role](https://github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket-replication-role)   | Module for creating an IAM role for S3 bucket replication                                                                                                       |
+| [modernisation-platform-terraform-aws-vm-import](https://github.com/ministryofjustice/modernisation-platform-terraform-aws-vm-import)                             | Module that creates s3 and roles needed to import virtual machine (VM) images from your virtualization environment to Amazon EC2 as Amazon Machine Images (AMI) |
+| [modernisation-platform-terraform-lambda-scheduler-stop-start](https://github.com/ministryofjustice/modernisation-platform-terraform-lambda-scheduler-stop-start) | Module for stopping and starting instance, rds resources and autoscaling groups with lambda function                                                            |
+| [modernisation-platform-terraform-pagerduty-integration](https://github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration)             | Module associating an SNS topic with a PagerDuty service                                                                                                        |
+| [modernisation-platform-terraform-loadbalancer](https://github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer)                               | Module that creates application loadbalancer in AWS with logging enabled, s3 to store logs and Athena DB to query logs                                          |
+| [modernisation-platform-github-oidc-provider](https://github.com/ministryofjustice/modernisation-platform-github-oidc-provider)                                   | Module for creating OIDC providers to use in GitHub Actions                                                                                                     |
```

Comment:
Could this be moved to the `Terraform modules - used by the core platform` section please as we wouldn't expect members to be directly using this module, thanks!